camel/messages/base.py

```diff
@@ -373,6 +391,108 @@ def to_openai_user_message(self) -> OpenAIUserMessage:
                 "content": self.content,
             }
 
+    def to_multimodal_model_user_message(self) -> OpenAIUserMessage:
+        r"""Converts the message to an :obj:`OpenAIUserMessage` object.
+
+        Returns:
+            OpenAIUserMessage: The converted :obj:`OpenAIUserMessage` object.
+        """
+        hybird_content: List[Any] = []
+        hybird_content.append(
+            {
+                "type": "text",
+                "text": self.content,
+            }
+        )
```

Comment:
```suggestion
        hybird_content: List[Any] = [{
                "type": "text",
                "text": self.content,
            }]
```

Also there is a small typo in the name: `hybird -> hybrid`